widgets/src/lib/Icons/IconImageToText.svelte

```diff
@@ -24,5 +24,6 @@
     <path
 		d="M12 30H6a2.002 2.002 0 0 1-2-2V4a2.002 2.002 0 0 1 2-2h16a2.002 2.002 0 0 1 2 2v10h-2V4H6v24h6z"
 		fill="currentColor"
+	
 	/>
 </svg>
```

Comment:
oh my bad, I meant a new line at the end of the file (github highlighting made it look like I was asking new line at line 27)

otherwis, lgtm!